rest/index.py

```diff
@@ -22,62 +24,83 @@
 from bson import json_util
 from bson.codec_options import CodecOptions
 
-from mozdef_util.elasticsearch_client import ElasticsearchClient, ElasticsearchInvalidIndex
+from mozdef_util.elasticsearch_client import (
+    ElasticsearchClient,
+    ElasticsearchInvalidIndex,
+)
 from mozdef_util.query_models import SearchQuery, TermMatch
 
 from mozdef_util.utilities.logger import logger, initLogger
 from mozdef_util.utilities.toUTC import toUTC
 
 
 options = None
-pluginList = list()   # tuple of module,registration dict,priority
+pluginList = list()  # tuple of module,registration dict,priority
+
+# The name of the MongoDB database that stores duplicate chains.
+DUP_CHAIN_DB = "duplicatechains"
+
+# The format string that UTC-based timestamps are encoded as.
+DUP_CHAIN_DATE_FMT = "%Y/%m/%d %H:%M:%S"
```

Comment:
We should use https://github.com/mozilla/MozDef/blob/master/mozdef_util/mozdef_util/utilities/toUTC.py#L10 instead of needing to define the format like this.